docker/runtime/nodejs/kubeless.js

```
@@ -70,22 +71,38 @@ function modExecute(handler, req, res, end) {
         throw new Error(`Unable to load ${handler}`);
 
     try {
-        let data = req.body;
+        let event;
+        let cType = contentType.parse(req);
         if (req.body.length > 0) {
-            if (req.get('content-type') === 'application/json') {
-                data = JSON.parse(req.body.toString('utf-8'))
-            } else {
-                data = req.body.toString('utf-8')
+            if (cType.type.startsWith('application/cloudevents')){
+                if (cType.type.endsWith('+json')){
+
+                    event = JSON.parse(req.body.toString('utf-8'));
+                    console.log('Event: '+req.body.toString());
+                }
+                else {
+//  We only support +json right now. Throw an error?
```

yes, I would throw an error here saying that the content-type is not supported.